internal/pkg/docker/dockercompose/decompose.go

```diff
@@ -122,3 +133,32 @@ func unsupportedServiceKeys(service map[string]any, svcName string) (IgnoredKeys
 
 	return ignored, nil
 }
+
+// isolateEnvFiles manually handles the env_file key so that compose-go does not try to read the env files from the disk.
+func isolateEnvFiles(service map[string]interface{}) ([]string, error) {
+	envFileStr := struct {
+		EnvFile string `mapstructure:"env_file"`
+	}{}
+	err := mapstructure.Decode(service, &envFileStr)
+
+	if err == nil && envFileStr.EnvFile != "" {
+		return []string{envFileStr.EnvFile}, nil
+	}
+
+	envFileOnly := struct {
```

Comment:
would this var name make it a little clearer that the above is for
```
env_file: .env
```
while this is for
```
env_file:
  - ./a.env
  - ./b.env
```
?
```suggestion
	envFileList := struct {
```

Reply:
Good idea, done!